desktop-widgets/tab-widgets/maintab.cpp

```diff
@@ -309,9 +305,6 @@ void MainTab::divesChanged(const QVector<dive *> &dives, DiveField field)
 	case DiveField::VISIBILITY:
 		ui.visibility->setCurrentStars(current_dive->visibility);
 		break;
-	case DiveField::SUIT:
```

Comment:
You remove the code here, but don't add it for the equipment-tab -> undo/redo will not work properly.

Reply:
This creates some more code sharing with maintab.cpp, but we have already discussed this.